tests/test-firefox/test.remote.js

```diff
@@ -172,6 +172,7 @@ describe('firefox.remote', () => {
         const stubResponse = {requestTypes: ['reload']};
 
         const conn = makeInstance();
+        // $FLOW_IGNORE: override class method for testing reasons.
```

Comment:
I think when designing this type of thing in the future, we should not override properties but instead use a class instance variable to allow injection (or something like that).